Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:8.1
```

Comment:
Was this originally done as part of the attempt to get a new PHP 8 (and/or 8.1) Docker image ready for use in our apps? If so, I'm wondering if we can just shift this back to using our PHP Dockerfile (whether our 7.4 image or a new 8.x image based off our 7.4 image, instead of based of the official PHP images).